app/javascript/lib/validation/chunked-line-reader.js

```diff
@@ -20,9 +20,9 @@ export default class ChunkedLineReader {
     this.linesRead = 0
     this.currentFragment = null // currentFragment stores the parts of lines that cross chunk boundaries
     this.chunkLines = []
+    this.startTime = new Date().getTime() // start the CSFV timer
```

Comment:
For consistency with faster timing method:

```suggestion
    this.startTime = Date.now() // start the CSFV timer
```